internal/pkg/describe/env.go

```diff
@@ -246,6 +253,27 @@ func (d *EnvDescriber) filterDeployedSvcs() ([]*config.Workload, error) {
 	return deployedSvcs, nil
 }
 
+// filterDeployedJobs lists the jobs that are deployed on the given app and environment
+func (d *EnvDescriber) filterDeployedJobs() ([]*config.Workload, error) {
+	allJobs, err := d.configStore.ListJobs(d.app)
+	if err != nil {
+		return nil, fmt.Errorf("list services for app %s: %w", d.app, err)
+	}
+	jobs := make(map[string]*config.Workload)
+	for _, job := range allJobs {
+		jobs[job.Name] = job
+	}
+	deployedJobNames, err := d.deployStore.ListDeployedJobs(d.app, d.env.Name)
```

Comment:
hmm we don't necessarily address this in this particular PR but in the near future we should refactor this so that `listDeployedWorkloads` of `deployStore` returns `[]*config.Workload` instead of `[]string`. This will simplify the logic of `filterDeployedSvcs` and `filterDeployedJobs`.

Reply:
yes totally agree, I guess then we don't need to make 2 separate calls to bring `[]*config.Workload` and then `[]string` and add custom logic to filter them.